pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -130,65 +170,65 @@ func mergeKeys(key1 []byte, key2 []byte) ([]byte, error) {
 	mergedKey := []byte("")
 	mergedKey = append(mergedKey, key1[0:v1]...)
 	mergedKey = append(mergedKey, key2[v1:v2]...)
+	log.Info("Merging keys")
 	return mergedKey, nil
 }
 
-func deriveVaultKey() ([]byte, error) {
+func deriveVaultKey(oldKeyMode bool) ([]byte, error) {
```

Comment:
Would it make sense to add a comment that oldKeyMode is set when the key is only using the cloud and not the TPM?

If so, is it really "old" or do we want to have a mode without a TPM when we use only the cloud key?
Aka "oldKeyMode" maybe should be named "cloudKeyOnlyMode"??

Reply:
Renamed oldKeyMode to cloudKeyOnlyMode, and added a comment as well.